orgprobe

```diff
@@ -41,22 +41,7 @@ loaded = config.read(args.config)
 
 logging.info("Loaded %s config files from %s", loaded, args.config)
 
-if config.has_section('api'):
-    def apiconfig(prop, method):
-        try:
-            setattr(APIRequest, prop.upper(), method('api', prop))
```

Comment:
Using setattr in this way made unit tests for the probe a bit unpleasant, hence the overhaul to the api interfacing code.